sql/common/src/test/scala/org/apache/sedona/sql/dataFrameAPITestScala.scala

```diff
@@ -957,5 +957,13 @@ class dataFrameAPITestScala extends TestBaseScala {
       val expectedResult = 3
       assert(actualResult == expectedResult)
     }
+
+    it("Passed ST_Force3D") {
+      val lineDf = sparkSession.sql("SELECT ST_Force3D(ST_GeomFromWKT('LINESTRING (0 1, 1 0, 2 0)'), 2.3) AS geom")
```

Comment:
Please add the test case that has no zvalue supplied.

Reply:
Added

Reply:
What I mean is, add a test case like ST_Force3D(geom), no z value is supplied. This will test the correctness of the the optional parameter.

Reply:
In addition, the test cases didn't test the correctness of the DataFrame style API, it mistakenly repeated the SQL test cases. Please read other test cases in this file and learn how to test DataFrame style APIs.

Explanation of Sedona DataFrame style APIs: https://sedona.apache.org/latest-snapshot/api/sql/DataFrameAPI/